meta/AccessTable.php

```diff
@@ -207,14 +207,20 @@ protected function consolidateData($DBdata, $asarray = false) {
             // if no data saved yet, return empty strings
             if($DBdata) {
                 $val = $DBdata[0]['out' . $col->getColref()];
+
+                // multi val data is concatenated
+                if($col->isMulti()) {
+                    $val = explode($sep, $val);
+                    $val = array_filter($val);
+                }
             } else {
-                $val = '';
-            }
+                $val = $col->getType()->getDefaultValue();
 
-            // multi val data is concatenated
-            if($col->isMulti()) {
-                $val = explode($sep, $val);
-                $val = array_filter($val);
+                // multi val data is concatenated
+                if($col->isMulti()) {
+                    $val = explode(',', $val);
+                    $val = array_map('trim', $val);
+                }
```

Comment:
It seems the else branch logic changed here. The isMulti() check used to be outside the else, now it is inside?